docs/application/dotnet/guides/nui/nui-components/RadioButton.md

```diff
@@ -0,0 +1,78 @@
+# RadioButton
+RadioButton is a common component and describes what action will occur when you select it.  
+A RadioButton can only contain an icon, and can be created using style.
+
+![RadioButton1](./media/RadioButton1.png)
+
+## Create with Style
+
+To create a RadioButton applying style, follow these steps:
```

Comment:
Suggest changing to:

To create a radio button using style, follow these steps:

Reply:
fixed